src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -488,7 +507,7 @@ private static ILogManager getLogManager(final PostgresSqlDb database,
     @Singleton
     private static ContentMapper getContentMapper() {
         if (null == mapper) {
-            mapper = new ContentMapper(getReflectionsClass("uk.ac.cam.cl.dtg.segue"));
+            mapper = new ContentMapper(getReflectionsClass("uk.ac.cam.cl.dtg"));
```

Comment:
This ensures the mapper _always_ knows about both `segue` and `isaac` packages, and is the first step in getting rid of the useless distinction.